### DISCUSSION TITLE
Change main menu

### DISCUSSION THREAD

**Discussion by andy5995** [netpanzer/netpanzer#96] [General]
I'm not fond of this bright red screen.  Are there plans for this screen? Like an image? Or can we reduce the brightness of that REDness?

![image](https://github.com/netpanzer/netpanzer/assets/16764864/6f9ea7ab-34bf-4c1f-9d04-c35a28ef95a7)

**Comment by winrid** [comment]
@ph2708 had tried to update it, but for some reason it only changed the loading screen. It should be changed.

**Comment by andy5995** [comment]
Issue created https://github.com/netpanzer/netpanzer/issues/97
